DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "749b0ddf",
+   "metadata": {},
+   "source": [
+    "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\"> \n",
+    "<br><b>CET Template Notebook</b> <br>\n",
+    "Contact author(s): <i>Sarah Greenstreet</i> <br>\n",
+    "Last verified to run: <i>2024-03-05</i> <br>\n",
+    "LSST Science Pipelines version: Weekly <i>2024_04</i> <br>\n",
+    "Container Size: <i>medium</i> <br>\n",
+    "Targeted learning level: <i>beginner</i> <br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9da1a210-d858-42fe-8591-570965b8be1a",
+   "metadata": {},
+   "source": [
+    "**Description:** _An introduction to near-Earth objects (NEOs) in the DP0.3 dataset, including illustrations of the four NEOs sub-populations: Amors, Apollos, Atens, and Atiras._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80a0baf5-51ad-40ec-8991-060a7b27c289",
+   "metadata": {},
+   "source": [
+    "**Skills:** _Use TAP queries to retrieve Solar System objects. Plot properties and orbits of near-Earth objects._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "393da88f-7978-4920-aa4a-a9830df6eed9",
+   "metadata": {},
+   "source": [
+    "**LSST Data Products:** _TAP DP0.3 MPCORB (10-year) and SSObject tables._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c67fab9-136a-4adc-bb42-142b91ab69dd",
+   "metadata": {},
+   "source": [
+    "**Packages:** _lsst.rsp.get_tap_service_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f72b27f",
+   "metadata": {},
+   "source": [
+    "**Credit:**\n",
+    "_Originally developed by Sarah Greenstreet and the Rubin Community Science Team in the context of the Rubin DP0._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28e91cbf-ab7f-4e26-9276-b00299d6065e",
+   "metadata": {},
+   "source": [
+    "**Get Support:**\n",
+    "Find DP0-related documentation and resources at <a href=\"https://dp0.lsst.io\">dp0.lsst.io</a>. Questions are welcome as new topics in the <a href=\"https://community.lsst.org/c/support/dp0\">Support - Data Preview 0 Category</a> of the Rubin Community Forum. Rubin staff will respond to all questions posted there."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfc73be0",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "_This tutorial demonstrates how to use TAP to query the MPCORB and SSObject tables. The tables are queried simultaneously and joined on their SSObjectId. The properties of near-Earth objects (NEOs; perihelia $q$<1.3 au) from the DP0.3 catalogs are examined. The NEOs in this sample are restricted to those with semimajor axes $a$<4.0 au, and eccentricities $e$<1.0. The orbital properties of NEOs are examined, including plotting the planet-crossing regions and boundaries that define the four NEO sub-populations: Amors, Apollos, Atens, and Atiras._"
```

Comment:
this is rendered as italic.  is that what we want for this section?

Reply:
Italics removed.